snippets/ember.cson

```diff
@@ -68,6 +68,11 @@
     'leftLabel': 'Ember'
     'body': 'inject.service(${1}),'
 
+  'service(...)':
```

Comment:
I think we should change `inject.service` instead of adding a new one. More in #43